exercises/Daniel_Murphy/ex02_oo_basics/src/shape.h

```diff
@@ -0,0 +1,103 @@
+#ifndef MAIN_H
+#define MAIN_H
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <cmath>
+#include <algorithm>
+#include <memory>
+#include <map>
+
+class Shape
+{
+public:
+  std::string m_type;
+  int m_sides;
+  int m_ID;
+
+  std::string get_type();
+  int get_sides();
+  int get_ID();
+
+
+  Shape(std::string type, int sides, int ID)
+    : m_type{type}, m_sides{sides}, m_ID{ID}
+  {}
```

Comment:
Please can you move the constructor to the top of the public section above any normal methods such as get_type